internal/handler/queue.go

```diff
@@ -0,0 +1,48 @@
+package handler
+
+import (
+	"encoding/json"
+	"log"
+    "fmt"
+	"net/http"
+
+	"github.com/SimplQ/simplQ-golang/internal/models"
+	"github.com/SimplQ/simplQ-golang/internal/persistence"
+)
+
+var Store persistence.QueueStore
```

Comment:
Instead of passing down the store to the mux from main, and then setting it there, let just initialise here? 

```
const store persistence.QueueStore = datastore.MongoDB()
```

Reply:
Can we change this separately after this PR? Calling `NewMongoDB` requires a URI which needs to be initialized somewhere.

I was thinking of creating a global config structure with the help of [godotenv](https://github.com/joho/godotenv)

Reply:
Let's hardcode the URL in the function, we can then move to reading it from some config / env / what you have in mind, as a follow up PR.